scripts/config-searchengine.js

```diff
@@ -0,0 +1,10 @@
+import {} from '../src/index.js'
+import SearchEngine from '../src/services/elasticsearch/index.js'
+
+async function run() {
```

Comment:
what is this script doing ?

Reply:
Previously, i config/init the search engine at the time that we boot up the code
But i tought it might be good practice to put them in a seperate script